internal/meta/data.go

```diff
@@ -20,6 +20,7 @@ type Data struct {
 	Title        string        `meta:"Title"`
 	Subject      string        `meta:"Subject,PersonInImage,ObjectName,HierarchicalSubject,CatalogSets"`
 	Keywords     Keywords      `meta:"Keywords"`
+	Favorite     bool          `meta:"FStopFavorite"`
```

Comment:
Does Exiftool support "FStopFavorite"? Otherwise the "meta" annotation isn't needed.

Reply:
Actually it's `Favorite` there:
```
$ exiftool fstop-favorite.xmp
ExifTool Version Number         : 12.30
File Name                       : fstop-favorite.xmp
Directory                       : .
File Size                       : 336 bytes
File Modification Date/Time     : 2021:03:15 22:06:40+01:00
File Access Date/Time           : 2022:01:02 14:23:21+01:00
File Inode Change Date/Time     : 2022:01:02 12:52:08+01:00
File Permissions                : -rw-rw-rw-
File Type                       : XMP
File Type Extension             : xmp
MIME Type                       : application/rdf+xml
XMP Toolkit                     : XMP Core 4.4.0-Exiv2
Favorite                        : 1
```

So should I change it like this?
```suggestion
	Favorite     bool          `meta:"Favorite"`
```